frontend/src/App.tsx

```diff
@@ -30,148 +31,7 @@ import WorldMap from './components/world/WorldMap';
 import CoveyAppContext from './contexts/CoveyAppContext';
 import NearbyPlayersContext from './contexts/NearbyPlayersContext';
 import VideoContext from './contexts/VideoContext';
-import { CoveyAppState, NearbyPlayers } from './CoveyTypes';
```

Comment:
needed to move reducers into a separate file to test, otherwise the canvas component for the game would cause jest crashes